Index: imx8mqevk/workspace/sources/chromium-ozone-wayland/content/gpu/BUILD.gn
===================================================================
--- imx8mqevk.orig/workspace/sources/chromium-ozone-wayland/content/gpu/BUILD.gn
+++ imx8mqevk/workspace/sources/chromium-ozone-wayland/content/gpu/BUILD.gn
@@ -133,11 +133,4 @@ target(link_target_type, "gpu_sources")
   if (enable_vulkan) {
     deps += [ "//gpu/vulkan" ]
   }
-
-  # Use DRI on desktop Linux builds.
-  if (current_cpu != "s390x" && current_cpu != "ppc64" &&
-      (is_linux || is_chromeos_lacros) &&
-      (!is_chromecast || is_cast_desktop_build)) {
-    configs += [ "//build/config/linux/dri" ]
-  }
 }
